site/_docs/troubleshooting.md

```diff
@@ -60,12 +60,43 @@ which may allow you to install native gems using this command (again using
 sudo gem install jekyll
 {% endhighlight %}
 
-Note that upgrading MacOS X does not automatically upgrade Xcode itself
+Note that upgrading Mac OS X does not automatically upgrade Xcode itself
 (that can be done separately via the App Store), and having an out-of-date
 Xcode.app can interfere with the command line tools downloaded above. If
 you run into this issue, upgrade Xcode and install the upgraded Command
 Line Tools.
 
+### Jekyll &amp; Mac OS X 10.11
+
+With the introduction of System Integrity Protection, several directories
+that were previously writable are now considered system locations and are no
+longer available. As a result, it is recommended that you choose one of a
+number of available Ruby environments ([RVM][], [rbenv][], [chruby][], etc.) in
+which to install Jekyll.
+
+[RVM]: https://rvm.io
+[rbenv]: http://rbenv.org
+[chruby]: https://github.com/postmodern/chruby
+
+If you elect to use a method other than Homebrew to install Ruby, it may be
```

Comment:
Home brew seems to come out of the blue here. Is it mentioned above? I'm on mobile and can't check easily.

Additionally, we could say "other than Homebrew to install Ruby, such as those listed above, ..." To clarify that they are ruby managers